core/http/api/v1/cluster/routers.go

```diff
@@ -23,125 +23,129 @@ import (
 	"github.com/horizoncd/horizon/pkg/server/route"
 )
 
-// RegisterRoutes register routes
-func (api *API) RegisterRoute(engine *gin.Engine) {
+// RegisterRoute register routes
+func (a *API) RegisterRoute(engine *gin.Engine) {
 	apiGroup := engine.Group("/apis/core/v1")
 	var routes = route.Routes{
 		{
 			Method:      http.MethodPost,
 			Pattern:     fmt.Sprintf("/applications/:%v/clusters", common.ParamApplicationID),
-			HandlerFunc: api.Create,
+			HandlerFunc: a.Create,
```

Comment:
any reason why changed api to a in this file ??
is api keyword already used in this file ?

Reply:
for consistency with the receiver in api.go